### ISSUE TITLE
Remove brackets when composing attribute/methods calls

### ISSUE THREAD

**Issue by mathieutu** [prettier/plugin-php#112]
```diff
-        return $this->quantity * $this->{self::PRICE_ATTRIBUTE};
+        return $this->quantity * $this->self::PRICE_ATTRIBUTE;
```

**Comment by czosel** [comment]
Also:
```php
$this->view->bar(
    $this->view->form->{$data['data-some-looooooooooooooooooooong-reference']}->getValue()
);
```
becomes
```php
$this->view->bar(
    $this->view->form->$data['data-some-looooooooooooooooooooong-reference'
    ]->getValue()
);
```

**Comment by czosel** [comment]
Fixed in #183
